src/app/contextBroker/contextBroker.cpp

```diff
@@ -315,6 +317,8 @@ bool            relogAlarms;
 #define STAT_NOTIF_QUEUE       "enable thread pool notifications queue statistics"
 #define LOG_SUMMARY_DESC       "log summary period in seconds (defaults to 0, meaning 'off')"
 #define RELOGALARMS_DESC       "log messages for existing alarms beyond the raising alarm log message itself"
+#define CHECK_v1_ID_DESC       "additional checks for ids and types in v1"
```

Comment:
"additional checks for id fields in the NGSIv1 API"

(Note that "ids and types" wouldn't cover attribute and metadata name)